### PR TITLE
Initial version of component-tracker card

### PR DESCRIPTION
**Pull request by ludeeus**
Add an initial version of a component tracker

### CODE REVIEW

File: card-tracker/changelog.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.0.4
+Changed defaul name
```
Comment (ciotlosm) [inline card-tracker/changelog.md]: spelling

File: component-tracker/component-tracker.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  setConfig(config) {
+    if (!config.entity) {
+      throw new Error('Please define an entity');
+    }
+
+    const root = this.shadowRoot;
+    if (root.lastChild) root.removeChild(root.lastChild);
+
+    const cardConfig = Object.assign({}, config);
+    if (!cardConfig.title) cardConfig.title = 'ðŸ“£ Custom Component Updates';
```
Comment (ciotlosm) [inline component-tracker/component-tracker.js]: Wrong character (emoticon)

**Comment by ciotlosm** [comment]
I can rename card tracker in tracker and make it work for both use cases. What do you think?

**Comment by ludeeus** [comment]
sounds good :)

**Pull request closed without merging**